src/components/GuaranteedAnalysisForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { FormComponentProps, LabelData, UNITS } from "@/types/types";
+import { Box, Typography } from "@mui/material";
+import { useEffect } from "react";
+import { FormProvider, useForm, useWatch } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+import VerifiedBilingualTable from "./VerifiedBilingualTable";
+import { VerifiedCheckbox, VerifiedInput } from "./VerifiedFieldComponents";
+
+function GuaranteedAnalysisForm({
+  labelData,
+  setLabelData,
+}: FormComponentProps) {
+  const { t } = useTranslation("labelDataValidationPage");
```

Comment:
Use arrow syntax here so it's consistent with the rest of the codebase. 
```JavaScript
const GuaranteedAnalysisForm = ({
  labelData,
  setLabelData,
}: FormComponentProps) => {
//code
}
```

Reply:
done